generator/views/datagroups.py

```diff
@@ -15,11 +15,11 @@
 
 DEFAULT_MAX_CACHE_AGE = "24 hours"
 
-# Note: INFORMATION_SCHEMA.PARTITIONS has a row with a `last_modified_time` value even for non-partitioned tables.
```

Comment:
I suspect yes, but does this also apply for `INFORMATION_SCHEMA.TABLE_STORAGE`? We probably still want to create datagroups for non-partitioned tables.

Reply:
That makes sense to me 👍 no need to re-add